doc/minetestserver.6

```diff
@@ -1,4 +1,4 @@
-.\" Minetestserver man page
+.\" Minetest-server man page
```

Comment:
Personally I'd just remove the line completely. It's not exactly a very useful comment considering the filename and also the contents of the file make it abundantly clear what it is.